examples/01_plotting/plot_demo_more_plotting.py

```diff
@@ -127,7 +130,8 @@
 
 plotting.plot_stat_map(stat_img, display_mode='z',
                        cut_coords=1, colorbar=False,
-                       title="display_mode='z', cut_coords=1, colorbar=False")
+                       title="display_mode='z', cut_coords=1, colorbar=False",
```

Comment:
Does it work if you put the title over 2 lines?

```suggestion
                       title="display_mode='z', cut_coords=1,\ncolorbar=False",
```

![image](https://github.com/nilearn/nilearn/assets/6961185/29210637-a44e-45dc-9c60-21ae561e1974)